ruby-server-example/bsf.hcl

```diff
@@ -0,0 +1,16 @@
+
+packages {
+  development = ["coreutils-full@9.5", "bash@5.2.15"]
```

Comment:
Rust development dependencies need to be added here which can later be shown in the development shell.

Reply:
you mean ruby*